resources/content/schema/admin/admin-auth.json

```diff
@@ -49,45 +73,60 @@
         "container.directory" : "cr",
         "container.dns" : "cr",
         "container.dnsSearch" : "cr",
-        "container.dockerHostIp" : "",
-        "container.dockerIp" : "",
-        "container.dockerPorts" : "",
+        "container.domain" : "cr",
         "container.domainName" : "cr",
         "container.entryPoint" : "cr",
         "container.environment" : "cr",
+        "container.firstRunning" : "r",
         "container.hostname" : "cr",
-        "container.imageId" : "cr",
         "container.imageUuid" : "cr",
+        "container.imageId" : "",
         "container.instanceLinks" : "cr",
         "container.kind" : "r",
         "container.lxcConf" : "cr",
         "container.memory" : "cr",
         "container.memorySwap" : "cr",
         "container.networkIds" : "cr",
-        "container.offeringId" : "",
+        "container.ports" : "cr",
+        "container.primaryIpAddress" : "r",
         "container.privileged" : "cr",
         "container.publishAllPorts": "cr",
+        "container.requestedHostId" : "cr", 
         "container.restartPolicy" : "cr",
         "container.startOnCreate" : "cr",
         "container.stdinOpen" : "cr",
         "container.subnetIds" : "cr",
+        "container.token" : "r",
         "container.tty" : "cr",
         "container.user" : "cr",
+        "container.userdata": "",
         "container.uuid" : "r",
-
+        "container.validHostIds" : "",
+        "container.vnetIds" : "",
+        "container.zoneId" : "",
+        
+        "containerExec" : "cr",
         "containerExec.attachStdin" : "cr",
         "containerExec.attachStdout" : "cr",
         "containerExec.tty" : "cr",
         "containerExec.command" : "cr",
-
-        "credential.publicValue" : "cru",
-        "credential.secretValue" : "cru",
-
+        
+        "containerExecOutput.*" : "r",
+
+        "credential" : "cr",
```

Comment:
Calling out as suspicious that public, secret, and kind are cr. Same deal as apikey? Discussed with @ibuildthecloud and this is what is needed?